lua/lvim/core/cmp.lua

```diff
@@ -23,7 +23,7 @@ end
 M.methods.feedkeys = feedkeys
 
 ---when inside a snippet, seeks to the nearest luasnip field if possible, and checks if it is jumpable
----@param dir number 1 for forward, -1 for backward; defaults to 1
+---@param dir? number 1 for forward, -1 for backward; defaults to 1
 ---@return boolean true if a jumpable luasnip field is found while inside a snippet
 local function jumpable(dir)
```

Comment:
do we actually still need this btw? have you checked their new API since?  https://github.com/L3MON4D3/LuaSnip/wiki/Nice-Configs#only-jump-in-the-current-snippet

Reply:
I'll have to review, but at a glance: yes, because it likely doesn't support resuming the cursor to the nearest modification after leaving insert mode for an unfinished snippet.

Reply:
After reviewing the code, yes. It still is required but uses the updated code which simply calls `in_snippet()` and `jumpable()`, which is done at the end of the function.

Reply:
> I'll have to review, but at a glance: yes, because it likely doesn't support resuming the cursor to the nearest modification after leaving insert mode for an unfinished snippet.

how do you feel about upstreaming this API? I remember asking @L3MON4D3 about it a while back.

Reply:
After further testing, I've removed the code and deprecated the exported function :+1:

Reply:
> > I'll have to review, but at a glance: yes, because it likely doesn't support resuming the cursor to the nearest modification after leaving insert mode for an unfinished snippet.
> 
> how do you feel about upstreaming this API? I remember asking @L3MON4D3 about it a while back.

As stated above, shouldn't be necessary.